### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by kosher9**
Changes made in this pull request :
- [x] Created a method called current_user in your ApplicationController.
- [x] Created forms to perform the following functions:

    Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
    Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- [x] Allowed Users to add likes to Posts.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ To get a local copy up and running follow these simple example steps.
 # Create user
 sudo -u postgres createuser -e ced00 -s
 # Create the database
-rake db:create
+rake db:new
 # Create database Migration
 rails db:migrate
 ```
```
Comment (MahmoudBakr23) [inline README.md]: **Optional**
- Please add the author section and MIT license for a professional repo.

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @kosher9 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Nice job adding some styles to your app
- Great work making the necessary RESTful methods in the routes and controller
- Well done adjusting the views templates and adding the forms
- Well done adding the RSpec tests
- Correct Linters
- Descriptive PR summary and headline

**Optional**
_Please see the comment under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (kosher9) [reply README.md]: Hi @MahmoudBakr23 , please what do you mean by 'author section'?

Comment (kosher9) [reply README.md]: @MahmoudBakr23 It's fine. I get it now.

**Pull request merged**